distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -425,6 +425,9 @@
       Enable following config to allow Emails as usernames.
     -->
     <EnableEmailUserName>{{tenant_mgt.enable_email_domain | default('false')}}</EnableEmailUserName>
+    {% if password_trim.enable is defined %}
+      <EnablePasswordTrim>{{password_trim.enable}}</EnablePasswordTrim>
```

Comment:
Have you added the default value for this in default.json

Reply:
We don't need default value, Therefor we use if condition here